src/internal_events/prometheus.rs

```diff
@@ -171,18 +198,20 @@ impl InternalEvent for PrometheusNoNameError {
         error!(
             message = "Could not decode timeseries.",
             error = "Decoded timeseries is missing the __name__ field.",
+            error_type = "parse_failed",
             stage = "processing",
-            internal_log_rate_secs = 5
```

Comment:
Did you mean to drop the rate limit here?

Reply:
I forgot them, just added them all 😉